models/vista2d/docs/README.md

```diff
@@ -0,0 +1,165 @@
+## Overview
+
+The **VISTA2D** is a cell segmentation training and inference pipeline for cell imaging [[`Blog`](https://developer.nvidia.com/blog/advancing-cell-segmentation-and-morphology-analysis-with-nvidia-ai-foundation-model-vista-2d/)].
+
+A pretrained model was trained on collection of 15K public microscopy images. The data collection and training can be reproduced following the [tutorial](./download_preprocessor/). Alternatively, the model can be retrained on your own dataset. The pretrained vista2d model achieves good performance on diverse set of cell types, microscopy image modalities, and can be further finetuned if necessary.  The codebase utilizes several components from other great works including [SegmentAnything](https://github.com/facebookresearch/segment-anything)  and [Cellpose](https://www.cellpose.org/), which must be pip installed as dependencies.  Vista2D codebase follows MONAI bundle format and its [specifications](https://docs.monai.io/en/stable/mb_specification.html).
+
+<div align="center"> <img src="https://developer-blogs.nvidia.com/wp-content/uploads/2024/04/magnified-cells-1.png" width="800"/> </div>
+
+
+### Model highlights
+
+- Robust deep learning algorithm based on transformers
+- Generalist model as compared to specialist models
+- Multiple dataset sources and file formats supported
+- Multiple modalities of imaging data collectively supported
+- Multi-GPU and multinode training support
+
+
+### Generalization performance
+
+Evaluation was performed for the VISTA2D model with multiple public datasets, such as TissueNet, LIVECell, Omnipose, DeepBacs, Cellpose, and [more](./docs/data_license.txt). A total of ~15K annotated cell images were collected to train the generalist VISTA2D model. This ensured broad coverage of many different types of cells, which were acquired by various imaging acquisition types. The benchmark results of the experiment were performed on held-out test sets for each public dataset that were already defined by the dataset contributors. Average precision at an IoU threshold of 0.5 was used for evaluating performance. The benchmark results are reported in comparison with the best numbers found in the literature, in addition to a specialist VISTA2D model trained only on a particular dataset or a subset of data.
+
+<div align="center"> <img src="https://developer-blogs.nvidia.com/wp-content/uploads/2024/04/vista-2d-model-precision-versus-specialist-model-baseline-performance.png" width="800"/> </div>
+
+
+
+### Install dependencies
+
+```
+pip install monai fire tifffile imagecodecs pillow fastremap
+pip install --no-deps cellpose natsort roifile
```

Comment:
Hi @KumoLiu , we have some new dependency here.

Reply:
Hi @mingxin-zheng , I remove this section and update metadata.json.
If specifying suitable version, all these dependencies can be installed with `pip install` (no need --no-deps) directly